### PR TITLE
TYPE: Add py.typed

### LINKED ISSUE #36: TYPE: missing py.typed
From mypy:
```
error: Skipping analyzing "odc.geo": module is installed, but missing library stubs or py.typed marker
```

### PR DESCRIPTION
**Pull request by snowman2**
Closes #36

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -23,6 +23,7 @@ classifiers =
     Programming Language :: Python :: 3.9
     Topic :: Software Development :: Libraries :: Python Modules
     Topic :: Scientific/Engineering :: GIS
+    Typing :: Typed
```
Comment (Kirill888) [inline setup.cfg:26]: So this tells mypy that types are present?

File: setup.cfg [hunk 2]
```diff
@@ -56,6 +57,8 @@ include =
   odc*
 
 [options.package_data]
+odc.geo =
```
Comment (Kirill888) [inline setup.cfg:60]: And this to include any stubs?

~1. we currently don't have stubs, should they be generated at packaging?~
2. should this also go into MANIFEST.in for source distribution?

Comment (Kirill888) [reply setup.cfg:60]: Ok, I see, it's just an empty file marker.

Comment (snowman2) [reply setup.cfg:60]: > should this also go into MANIFEST.in for source distribution?

I haven't needed to. The package data part seems to do the trick.

File: setup.cfg (hunk 1 above)
Comment (snowman2) [reply setup.cfg:26]: https://mypy.readthedocs.io/en/stable/installed_packages.html

> If you would like to publish a library package to a package repository yourself (e.g. on PyPI) for either internal or external use in type checking, packages that supply type information via type comments or annotations in the code should put a py.typed file in their package directory.

**Pull request merged**